home-manager-module.nix

```diff
@@ -24,7 +24,7 @@
       enable = lib.mkDefault true;
       package = packages.${pkgs.stdenv.system}.nix-index-with-db;
     };
-    home.packages = lib.optional config.programs.nix-index.enable packages.${pkgs.stdenv.system}.comma-with-db;
+    home.packages = lib.optional config.programs.nix-index-database.enable packages.${pkgs.stdenv.system}.comma-with-db;
```

Comment:
This breaks the build: there's no such option `nix-index-database.enable`; did you mean `nix-index-database.comma.enable`?
Also, there're two sets of options, `programs.nix-index` and `programs.nix-index-database`, is this intended?